cdisc_rules_engine/services/reporting/excel_report.py

```diff
@@ -59,6 +60,23 @@ def get_export(
         )
         wb["Conformance Details"]["B3"] = f"{round(self._elapsed_time, 2)} seconds"
         wb["Conformance Details"]["B4"] = __version__
+
+        # write dataset metadata
+        datasets_data = [
+            [
+                dataset.get("filename"),
+                dataset.get("label"),
+                dataset.get("full_path"),
+                dataset.get("modification_date"),
+                dataset.get("size", 0) / 1000,
```

Comment:
should kb calculation be binary (1024) instead of decimal?

Reply:
I used decimal because I suspect our users will use decimal when converting the file size to kilobytes,  and I didn't want to cause any confusion.